src/simulator_core/infrastructure/app.py

```diff
@@ -57,7 +56,8 @@ def run(file_path: str | None = None) -> pd.DataFrame:
         raise error
 
 
-if __name__ == "__main__":
+def main() -> None:
```

Comment:
Are these modifications of app.py required in this PR?

Reply:
maybe not per se, but is clean up of the app.py code

Reply:
Removed it